README.md

```diff
@@ -13,7 +13,12 @@ cd /home/openfisca
 git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
-Each sub-directory has its own README.md file.
+For more information on `auto-update` sub-directories, please see their own README.md files.
+
+## Server configuration
+
+The main nginx services' configuration files are stored in a directory named after a server name.  
```

Comment:
- `nginx services` does not mean much : services are processes that are maintained alive by the OS. Nginx is just a reverse proxy that dispatch requests to the right application.
- `a server name` is ambiguous. Are we talking about a physical server ? You probably meant a domain name.